src/Templates/MyRadio/getOnAir.twig

```diff
@@ -9,7 +9,7 @@
 
 <!--Pay Membership-->
 <li>
-  <a href="{{ config.payment_url }}" target="_blank" title="Before we can let you use our shiny and expensive studio, you'll need to pay membership. Click here to do so on the YORKSU website now.">Become a Paid Member
+  <a href="{{ config.payment_url }}" target="_blank" title="Before we can let you use our shiny and expensive studio, you'll need to pay membership. Click here to do so on the YUSU website now.">Become a Paid Member
```

Comment:
This change can stay, although we probably want to use the official capitalisation
```suggestion
  <a href="{{ config.payment_url }}" target="_blank" title="Before we can let you use our shiny and expensive studio, you'll need to pay membership. Click here to do so on the York SU website now.">Become a Paid Member
```

Reply:
This was in the pending pr for nickname